### DISCUSSION TITLE
Cannot select "Leistung Heim" and "Leistung PV" in home assistants native section for "Energie"

### DISCUSSION THREAD

**Discussion by dklinger** [marq24/ha-evcc#14] [Q&A]
### Checklist

- [X] I have installed the **latest** release (or BETA) version of the integration and home assistant.
- [X] I have prepared DEBUG log output (for technical issues) | In most of the cases of a technical error/issue I would have the need to ask for DEBUG log output of the integration. There is a short [tutorial/guide 'How to provide DEBUG log' here](https://github.com/marq24/ha-senec-v3/blob/master/docs/HA_DEBUG.md)
- [X] I confirm it's really an issue | In the case that you want to understand the functionality of a certain feature/sensor Please be so kind and make use if the discussion feature of this repo (and do not create an issue) - TIA

### Add a description

Hi.
GRIDPOWER, HOMEPOWER and PVPOWER have unit_of_measuremen= UnitOfPower.WATT. 
I would like to use these sensors to power the native Energy functions of Home Assistant, which means I want to select these sensors under Energie -> Netzverbrauch and Engerie -> Netzeinspeisung in the Home Assistant UI.
But that's not possible as home assistant excepts the unit_of_measuremen to be an energy measurement, not a power measurement, for example UnitOfPower.WATTHOURS.
And idea on what I need to do to make these values available to be consumed by the Energy-area of home assistant?

![image](https://github.com/user-attachments/assets/57b15583-c6f0-4012-98e9-1fc8f76faa1d)

**Comment by marq24** [comment]
well - you can't change the world of Physics - Energy is Power per Time...

-> you can use the Riemann Sum Integal Platform to reach your goal... you can check out this as reference how to do it... -> https://github.com/marq24/ha-senec-v3/issues/4

**Comment by UDicke** [comment]
Hallo @dklinger, ich habe die Energiewerte direkt vom Netzzähler genutzt. Die stehen als Entity zumindest bei mir in Home Assitant zur Verfügung. Dafür ist die evcc Integration nicht nötig. Schau doch mal ob Du eine geeignete Quelle in Home Assistant bei Dir findest ohne das Du das Riemann Sum Integal benutzt.

Hello @dklinger, I used the energy values ​​directly from the network meter. At least for me, they are available as entities in Home Assitant. The evcc integration is not necessary for this. See if you can find a suitable source  in Home Assistant without using the Riemann Sum Integral.

![grafik](https://github.com/user-attachments/assets/55c5953c-97c9-4c74-892c-2e1c7dbbf182)

**Reply by dklinger** [reply]
Besten Dank. Aber mein 40 Jahre alter Drehstromzähler taucht sicherlich nicht in Home Assistant auf :D
Als HA-Neuling war ich auch auf der falschen Fährte. Ich dachte HA braucht hier den jeweils aktuelen kW-Wert und nicht den absoluten Verbrauch. Nachdem der Denkfehler weg war, hab ich auch einen entsprechden Wert dafür gefunden, den ich per ModBus aus meinem EMS bekomme.
